src/horama_starter_fdsl.py

```diff
@@ -149,36 +150,29 @@ def init_maco_buffer(image_shape, std_deviation=1.0, dataloader=0):
 
     if dataloader == 0:
         magnitude_path = pathlib.Path(
-            "outputs/magnitude_FDSL.pth"
+            "outputs/magnitude_FDSL_test.pth"
```

Comment:
```suggestion
            "outputs/magnitude_FDSL_test.pth"
```
このパスは関数の引数になっているのが理想的ですね。